### PR TITLE
Adds AdminService and functions for updating LDAP mappings

### PR DESCRIPTION
**Pull request by alindeman**
/cc: @sr

### CODE REVIEW

File: github/github.go [hunk 1]
```diff
@@ -188,6 +189,7 @@ func NewClient(httpClient *http.Client) *Client {
 
 	c := &Client{client: httpClient, BaseURL: baseURL, UserAgent: userAgent, UploadURL: uploadURL}
 	c.common.client = c
+	c.Admin = (*AdminService)(&c.common)
```
Comment (gmlewis) [inline github/github.go]: nit: please move this one line below c.Activity to keep these sorted.

File: github/github.go [hunk 2]
```diff
@@ -116,6 +116,7 @@ type Client struct {
 	common service // Reuse a single struct instead of allocating one for each service on the heap.
 
 	// Services used for talking to different parts of the GitHub API.
+	Admin          *AdminService
```
Comment (gmlewis) [inline github/github.go]: nit: please move this one line below Activity to keep these sorted.

**Comment by alindeman** [comment]
@gmlewis Cheers, good idea. I've pushed a commit to put it in the correct alphabetical order.

**Comment by gmlewis** [comment]
Looks great, @alindeman - thank you!
Awaiting second LGTM before merging.

File: github/admin.go [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+// AdminService handles communication with the admin related methods of the
+// GitHub API. These API routes are normally only accessible for GitHub
+// Enterprise installations.
+//
+// GitHub API docs: https://developer.github.com/v3/enterprise/
+type AdminService service
+
+// TeamLDAPMapping represents the mapping between a GitHub team and an LDAP
+// group.
```
Comment (shurcooL) [inline github/admin.go]: Minor thought. If it were me, I wouldn't create a second line just for one word. [Why not](https://twitter.com/rob_pike/status/563801489190043648) keep it together with the rest of the sentence?

Either way is completely fine.

File: github/admin.go [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+package github
+
+import "fmt"
+
+// AdminService handles communication with the admin related methods of the
+// GitHub API. These API routes are normally only accessible for GitHub
+// Enterprise installations.
+//
+// GitHub API docs: https://developer.github.com/v3/enterprise/
```
Comment (shurcooL) [inline github/admin.go]: Should this URL be https://developer.github.com/v3/enterprise/ldap/? That seems to be the specific page for the 2 endpoints being supported by `AdminService` here.

Unless the plan is to expand `AdminService` to cover all other parts of https://developer.github.com/v3/enterprise/, including https://developer.github.com/v3/enterprise/admin_stats/, https://developer.github.com/v3/enterprise/license/, https://developer.github.com/v3/enterprise/management_console/, etc.?

Comment (alindeman) [inline github/admin.go]: My intention was to add more routes from the enterprise/admin section at a later point, yep. There are other routes in the `admin/` namespace that I haven't added yet.

File: github/admin.go (hunk 3 above)
Comment (alindeman) [inline github/admin.go]: This is my editor trying to keep everything under 80. I should probably make it be a bit more liberal. Pushed an update for this :)

**Comment by gmlewis** [comment]
Thanks for the LGTM, @shurcooL, and @alindeman for the tweaks!
Merging.

**Comment by gmlewis** [comment]
Included in https://github.com/google/go-github/commit/69894f9ef7352c33b5d874b2e3f799424c0f7387

File: github/admin.go (hunk 3 above)
Comment (shurcooL) [inline github/admin.go]: Thanks. I noticed you've changed 2 instances of this for the method docs, but not the 2 instances for types. Just FYI, fixing that is very optional.

**Pull request closed without merging**